pkgs/by-name/hy/hydrapaper/package.nix

```
@@ -0,0 +1,61 @@
+{ lib,
+  stdenv,
+  fetchFromGitLab,
+  meson,
+  ninja,
+  python3,
+  blueprint-compiler,
+  gobject-introspection,
+  pandoc,
+  libadwaita,
+  glib,
+  pkg-config,
+  dbus,
+  appstream,
+  gtk3
+}:
+
+stdenv.mkDerivation {
+  pname = "hydrapaper";
+  version = "3.3.2";
+
+  src = fetchFromGitLab {
+    owner = "gabmus";
+    repo = "HydraPaper";
+    rev = "4589088da390362906ac9fecb8d4b26f594656d9";
+    hash = "sha256-IDaM8bM/0KH9h59523WqLKe400V5lLNyJ4faPf980Ro=";
+  };
+
+  nativeBuildInputs = [
+    blueprint-compiler
+    gobject-introspection
+    meson
+    pandoc
+    ninja
+    glib
+    pkg-config
+    dbus
+    appstream
+    gtk3
```

```suggestion
    gtk4
    wrapGAppsHook
```
The dependency listed in the source repo is `gtk4`.
Include wrapGAppsHook to fix `ValueError: Namespace Gtk not available`.